src/coreclr/jit/llvmlower.cpp

```diff
@@ -635,7 +662,9 @@ GenTreeCall::Use* Llvm::lowerCallReturn(GenTreeCall*      callNode,
         {
             indirNode = _compiler->gtNewIndir(callReturnType, returnAddrLclAfterCall);
         }
-        indirNode->gtFlags |= GTF_IND_TGT_NOT_HEAP; // No RhpAssignRef required
```

Comment:
can you explain why we dont need this anymore, thanks?

Reply:
The flag is only relevant for stores i. e. `STOREIND`. This is a use, so it has not effect.

Reply:
oh yes, thanks